src/main/resources/init_db.sql

```diff
@@ -18,3 +16,30 @@ CREATE TABLE `drivers` (
                                   PRIMARY KEY (`id`),
                                   UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
                                   UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
+CREATE TABLE `cars` (
+                                    `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+                                    `manufacturer_id` BIGINT(11) NOT NULL ,
+                                    `model` VARCHAR(255) NOT NULL,
+                                    `is_deleted` TINYINT(1) NOT NULL DEFAULT 0,
+                                    PRIMARY KEY (`id`),
+                                    CONSTRAINT `cars_manufacturers_fk`
+                                                                        FOREIGN KEY (`manufacturer_id`)
+                                                                            REFERENCES `taxi_service`.`manufacturers` (`id`)
+                                                                            ON DELETE NO ACTION
+                                                                            ON UPDATE NO ACTION);
+
+CREATE TABLE `cars_drivers` (
+                                `driver_id` BIGINT(11) NOT NULL,
+                                `car_id` BIGINT(11) NOT NULL,
+                                CONSTRAINT `cars_drivers_drivers_fk`
+                                                                        FOREIGN KEY (`driver_id`)
+                                                                            REFERENCES `taxi_service`.`drivers` (id)
+                                                                            ON DELETE NO ACTION
+                                                                            ON UPDATE NO ACTION,
+                                CONSTRAINT `cars_drivers_cars_fk`
+                                                                        FOREIGN KEY (`car_id`)
+                                                                            REFERENCES `taxi_service`.`cars` (id)
+                                                                            ON DELETE NO ACTION
+                                                                            ON UPDATE NO ACTION
+);
```

Comment:
dont forget about empty line in the end of file

Reply:
thank you, I wish maven had this among the stylechecks